main.html

```diff
@@ -1,77 +1,240 @@
-<h1>A Simple Oriole</h1>
-<p>A really simple example:</p>
-<p>You can write paragraphs like this.</p>
-<pre data-code-language="python" data-executable="true" data-type="programlisting">hey = &quot;hello world&quot;, 38+4
-print(hey hello. This is an executable code block )!
-</pre>
-<div id="the-id">
-<p>This is a block inside a div with an id the-id.
-Very cool. Yes.</p>
-</div>
-<pre data-code-language="" >hey = &quot;hello world&quot;, 38+4
-print(hey hello. This is a non executable block )!
-</pre>
-<div id="another-id">
-<h2>More complicated example</h2>
-</div>
-<pre data-code-language="python" data-executable="true" data-type="programlisting">x = [1,1]
-for i in range(12):
-    x.append(x[-1] + x[-2]) 
-print(', '.join(str(y) for y in x))
-</pre>
-<h2>A Chart</h2>
-<div id="this-example">
-</div>
-<pre data-code-language="python" data-executable="true" data-type="programlisting">%matplotlib inline
-import matplotlib.pyplot as plt
-import numpy as np
-x = np.linspace(0, 3*np.pi, 500)
-plt.plot(x, np.sin(x**2))
-plt.title('A simple chirp');
-</pre>
-<h2>Seaborn Example</h2>
-<pre data-code-language="python" data-executable="true" data-type="programlisting">from __future__ import print_function, division
+Where will we stop for feedback and assessment questions?
+
+<h2>Introduction</h2>
 
-%matplotlib inline
-import matplotlib.pyplot as plt
-import numpy as np
-import pandas as pd
+<p>
+    InnerSource is the methodology of applying open source practices and principles to internal software development.
+    This methodology provides a low-friction method for ensuring that the features and functionality of shared projects are responsive and nimble to the changing needs of their many consumers.
+</p>
 
-x = np.linspace(0, 10, 1000)
-plt.plot(x, np.sin(x), x, np.cos(x));
+<p>
+    In this Oriole we'll define some key terms used when discussing InnerSource.
+    We'll walk through the high-level mechanics of how InnerSourcing can work and enumerate the key principles upon-which these mechanics are based.
+    Finally, we'll share other references that can help you deepen your understanding and involvement in the craft of InnerSource.
+</p>
 
-import seaborn as sns
-sns.set()
-plt.plot(x, np.sin(x), x, np.cos(x));
+<h2>Problem</h2>
+
+<p>
+    Imagine separate teams producing separate pieces of software with one team's software depending on that of another.
+    This situation is common in a large enterprise where a single team producing software may have dozens or hundreds of consumers.
+    When consuming teams need changes, normally the producing team has some sort of requirements prioritization process to decide which they will work on.
+    For feature requests that are not priorized, the consuming team is faced with 3 choices, each of which come with their own drawbacks.
```

Comment:
It may not pop out from an initial reading of the document, but each of these options intentionally has 3 syllables with the same syllabic emphasis pattern.  Just my opinion, but I think that this symmetry makes them easy to speak and natural to substitute for one another in sentences just as the actions that they represent are, in some way, substitutes for each other in real life.